cmd/up/main.go

```diff
@@ -623,3 +623,31 @@ func scheduleHTTPServer(l log.Logger, opts options.Options, reg *prometheus.Regi
 		}
 	})
 }
+
+const (
```

Comment:
I think we could move the const up together with the other ones already defined here: https://github.com/observatorium/up/blob/master/cmd/up/main.go#L37

Reply:
👍